models/ONNX/model_checker.py

```diff
@@ -0,0 +1 @@
+# TODO: execute with  onnx runtime
```

Comment:
Not sure what should go here: execution through RedisAI as well as native and checking if they coincide?

Reply:
Yes, the idea is to execute the model with ONNXRuntime and make a way for users to compare how it will be executed inside RedisAI. let's merge this if it is good to be there for 0.3. I am making another PR with SparkML and will include ONNXRuntime stuff there.

Reply:
Works for me